### ISSUE TITLE
Update screenshots in documentation

### ISSUE THREAD

**Issue by lwrubel** [gwu-libraries/sfm-ui#1040]
After implementing Bootstrap 4 and accessibility fixes, screenshots should be updated to reflect SFM 2.3.0 release. See the User Guide in particular.

**Comment by kerchner** [comment]
Screenshots I've been able to locate:
- https://gwu-libraries.github.io/sfm-ui/about/overview
- https://sfm.readthedocs.io/en/stable/userguide.html#what-is-sfm-used-for

**Comment by omachala** [comment]
Hi! I can help update these documentation screenshots.

I maintain [heroshot](https://heroshot.sh), a screenshot automation tool that captures screenshots from a config file using Playwright. Makes it easy to keep docs current.

Happy to submit a PR!
